nixos/modules/services/web-servers/nginx/default.nix

```diff
@@ -442,6 +444,9 @@ let
         proxy_set_header Upgrade $http_upgrade;
         proxy_set_header Connection $connection_upgrade;
       ''}
+      ${optionalString (config.uwsgiPass != null)
+        "uwsgi_pass ${config.uwsgiPass};"
```

Comment:
Do we also need the `proxyResolveWhileRunning` workaround for uwsgi?

Reply:
I think it functions the same but I don't think we need that option at all. According to the nginx docs, the record is cached for it's TTL lifetime. If you want to refresh it more often, I would recommend to reduce the records TTL to a minumum or use a caching proxy like unbound and set an upper limit on TTL.

I am going to create a PR to remove the option.

Reply:
#347164